src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1952,9 +2082,13 @@ internal bool Execute
 #endif
             GetAssemblyPathInGac getAssemblyPathInGac,
             IsWinMDFile isWinMDFile,
-            ReadMachineTypeFromPEHeader readMachineTypeFromPEHeader
+            ReadMachineTypeFromPEHeader readMachineTypeFromPEHeader,
+            GetRootedPath getRootedPath = null
         )
         {
+            // Make this default option
+            getRootedPath ??= FileUtilities.NormalizePath;
```

Comment:
Not sure, what you mean by this. getRootedPath can be passed as an argument.

Reply:
Right, so if they provide something, it takes that. Otherwise, you default to null then immediately change that to FileUtilities.NormalizePath, whereas I was suggesting you could default to FileUtilities.NormalizePath and skip the null.

Reply:
Oh I have overlooked that in diff. Still I can't use this, since `FileUtilities.NormalizePath` is not compile time constant

Reply:
Good point